### ISSUE TITLE
0crat should only count business days

### ISSUE THREAD

**Issue by amihaiemil** [zerocracy/farm#634]
Most people do not work in the weekends, so I believe weekends should not be counted in the deadlines.
Or at least make this configurable per project.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@amihaiemil well, it's a valid idea, but I don't really like it. We are all about remote work with no offices, etc. A holiday is something we inherit from 5-day nine-to-five working schedule. Most freelancers, actually, do work on holidays more than on weekdays.

**Comment by amihaiemil** [comment]
@yegor256 

> A holiday is something we inherit from 5-days nine-to-five working schedule. 

Yes, but it's not that simple. Most of the world works only on week days. If I should work on the weekends then I lose time I could spend with friends/family etc, since they, unlike me, are working 9-to-5.

Besides, ZC is still far from a replacement of a full-time job or even traditional freelancing (which most of the times translates to 9-to-5). It is still a part-time (even less than part-time) thing, so we shouldn't be so strict, at least in the beginning. Or, again, it could be configurable/project, since the project decides how fast and how rmeote the work should actually be.

**Comment by yegor256** [comment]
@amihaiemil you almost convinced me... let's try to put this question to a vote in Telegram.

**Comment by g4s8** [comment]
@amihaiemil I think it is not right direction to say which days will be counted, I think it should be configured via `x` parameter in "x-days-rule", I mean if you think that 10 days is not enough to complete some amount of jobs, it should be extended to 11 or 12 days rule.

**Comment by amihaiemil** [comment]
@g4s8 no, 10 days are enough, but weekends should simply not be counted. I just want to enjoy my weekends without having to worry about some task that is still waiting for ARCH's review, or receive emails from 0crat: "task has been assigned to you for 8 days".

Plus, it's not only about me, I personally can ignore it, but others cannot -- the PRs' authors are spamming me in the weekends, for instance. And sometimes it may happen, people lose tasks, those 2 days when I don't do anything are affecting people, it is not correct.

**Comment by g4s8** [comment]
@amihaiemil I see. So maybe it should be configurable for your profile (not project)? Because other people (as me) are working on weekends and not working on some workdays, and I can't say exacly on which week days I'll work next month and on which not. I think can be implemented via "notification settings" in your profile where you can click checkbox "don't notify me on weekends" and you will not receive anything from 0crat on weekends

**Comment by amihaiemil** [comment]
@g4s8 yes, that could also work :)
